ete3/tools/ete_build_lib/task/cog_selector.py

```diff
@@ -143,9 +143,9 @@ def sort_cogs_by_sp_repr(c1, c2):
         cognumber += 1 # sets the ammount of cogs in file
         for sp, ncogs in sorted(list(sp2cogs.items()), key=lambda x: x[1], reverse=True):
             log.log(28, "% 20s  found in single copy in  % 6d (%0.1f%%) COGs " %(sp, ncogs, 100 * ncogs/float(cognumber)))
-
+        
         valid_cogs = sorted([sing for sing in all_singletons if len(sing) >= min_species],
-                            sort_cogs_by_size)
+                            sort_cogs_by_size) #Ziqi issue
```

Comment:
I remember this line was linked to a bug in Python 3. Is it solved?

Reply:
It's fixed now